fastn-core/src/auth/email_password/set_password.rs

```diff
@@ -216,6 +217,7 @@ pub(crate) async fn forgot_password_request_success(
     req.log_with_no_site(
```

Comment:
Remove the site business from here. This package will not know about site/org etc.